Source/ACE.Entity/Enum/ObjectType.cs

```diff
@@ -36,6 +36,17 @@ public enum ObjectType : uint
         LifeStone                  = 0x10000000,
         TinkeringTool              = 0x20000000,
         TinkeringMaterial          = 0x40000000,
-        Gameboard                  = 0x80000000
+        Gameboard                  = 0x80000000,
+
+        PortalMagicTarget                       = 268500992,
```

Comment:
I'm not a fan of mixing hex (for the earlier values), and decimal for these values of the enum.

Also, looking at these values, it appears that they are just combinations of the hex values (e.g. Weapon = MeleeWeapon | MissileWeapon).  They may be more clearly defined in that manner.